docs/pages/components/object-marker.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Object Marker
+id:
+keywords: object Marker, Marker, object, object-Marker, clickable
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/object-Marker.html
```

Comment:
in the .yml file url is 'object-marker.html' but here it is 'object-Marker.html'